app/components/certificate/description.tsx

```diff
@@ -0,0 +1,44 @@
+import { Flex, Heading, Text, HStack, VStack } from '@chakra-ui/react';
+
+interface DescriptionProps {
+  request: Boolean;
```

Comment:
What is the meaning of `request`?  The term is overloaded, and we probably need something more specific.